src/components/Navbar.jsx

```diff
@@ -71,16 +74,16 @@ const NavBar = ({ name, handleLogout, children, theme }) => {
             <Greeting>Hi, {name}</Greeting>
           </>
         )}
-        {theme.name === 'nwHacks' && (
+        {activeHackathon === 'nwhacks' && (
```

Comment:
I may be missing something, but is there a reason why there is no case for hackcamp LOL

Reply:
you're so right